lib/components/artwork_grids/generic_grid.js

```diff
@@ -129,6 +129,8 @@ const styles = StyleSheet.create({
 })
 
 const GenericArtworksGridContainer = Relay.createContainer(GenericArtworksGrid, {
+  shouldComponentUpdate: () => true,
```

Comment:
It can block containers from updating in certain cases: https://facebook.github.io/relay/docs/api-reference-relay-container.html#shouldcomponentupdate

So I thought maybe overriding it would encourage an update every time.